src/main/java/Duke.java

```diff
@@ -16,37 +15,76 @@ public static void main(String[] args) {
         System.out.println("_______________________________________________________");
         System.out.println("Your instructions, my Liege.");
         Information input = new Information();
+        Scanner myObj = new Scanner(System.in);
+        boolean condition = true;
         int counter = 0;
-        while (true) {
-            Scanner myObj = new Scanner(System.in);
+        while (condition) {
             String userInput = myObj.nextLine();
-            if (Objects.equals(userInput, "bye")) {
-                System.out.println("_______________________________________________________\n" + "Farewell, my Lord." + "\n" + "_______________________________________________________\n");
+            String instruction = userInput.contains(" ") ? userInput.substring(0, userInput.indexOf(" ")): userInput;
+            String instructionTask = userInput.contains(" ") ? userInput.substring(userInput.indexOf(" ") + 1) : userInput;
+
+            switch (instruction) {
+            case "bye":
+                System.out.println("Farewell, my Lord." + "\n" + "_______________________________________________________\n");
+                condition = false;
                 break;
-            } else if (Objects.equals(userInput, "list")) {
+            case "list":
                 for (int i = 0; i < 100; i++) {
                     if (input.List[i] == null) {
                         break;
                     } else {
-                        System.out.println((i+1) + ".[" + input.List[i].getStatusIcon() + "] " + input.List[i].description);
+                        System.out.println((i + 1) + ".[" + input.List[i].getTaskIcon() + "]" + "[" + input.List[i].getStatusIcon() + "] " + input.List[i].getDescription());
```

Comment:
Perhaps line may be too long, consider wrapping text on next line or breaking down to sub statements?